samples/net/aws_iot/boards/native_sim.conf

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# NEWLIB C
+CONFIG_NEWLIB_LIBC=n
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=n
+
+# Heap and stacks
+CONFIG_INIT_STACKS=y
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Networking config
+CONFIG_NET_NATIVE=y
+CONFIG_NET_IPV4=y
```

Comment:
These are missing from the other two `prj.conf`, is that intentional?

Reply:
no, can be removed everywhere